dev.fredol.open-tv.yaml

```diff
@@ -0,0 +1,1152 @@
+id: dev.fredol.open-tv
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: open_tv
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --filesystem=host:ro
+  - --socket=pulseaudio
+  - --filesystem=/tmp:rw
+  - --filesystem=xdg-pictures:rw
+  - --filesystem=xdg-videos:rw
+  - --filesystem=home/.var/app/io.mpv.Mpv/config/mpv/scripts:create
+  - --env=LC_NUMERIC=C
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --filesystem=xdg-run/pipewire-0:ro
+  - --own-name=org.mpris.MediaPlayer2.mpv.*
+  - --talk-name=org.gnome.SessionManager
+
+cleanup:
+  - '*.la'
+  - '*.a'
+
+environment:
+  WEBKIT_DISABLE_DMABUF_RENDERER: 1
+
+modules:
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/Fredolx/open-tv/releases/download/v1.0.0/flatpak.deb
+        sha256: 62555af870ee10e0e363bc457ca280046dc19094cbb4d96b7e5770931b28dcaf
+        only-arches: [x86_64]
+      - type: file
+        path: open_tv.desktop
+    build-commands:
+      - ar -x *.deb
+      - tar -xf data.tar.gz
+      - install -Dm755 usr/bin/open_tv /app/bin/open_tv
+      - install -Dm644 open_tv.desktop /app/share/applications/dev.fredol.open-tv.desktop
+      - install -Dm644 usr/share/icons/hicolor/128x128/apps/open_tv.png /app/share/icons/hicolor/128x128/apps/dev.fredol.open-tv.png
+      - install -Dm644 usr/share/icons/hicolor/32x32/apps/open_tv.png /app/share/icons/hicolor/32x32/apps/dev.fredol.open-tv.png
+      - install -Dm644 usr/share/icons/hicolor/256x256@2/apps/open_tv.png /app/share/icons/hicolor/256x256@2/apps/dev.fredol.open-tv.png
+  # MPV SECTION
+  - name: lmdb
+    sources:
+      - sha256: f3927859882eb608868c8c31586bb7eb84562a40a6bf5cc3e13b6b564641ea28
+        type: archive
+        url: https://github.com/LMDB/lmdb/archive/LMDB_0.9.22.tar.gz
+    make-install-args:
+      - prefix=/app
+    no-autogen: true
+    subdir: libraries/liblmdb
+
+  - name: parse-yapp
+    buildsystem: simple
+    build-commands:
+      - perl Makefile.PL INSTALL_BASE=/app
+      - make
+      - make install
+    sources:
+      - type: archive
+        url: https://cpan.metacpan.org/authors/id/W/WB/WBRASWELL/Parse-Yapp-1.21.tar.gz
+        sha256: 3810e998308fba2e0f4f26043035032b027ce51ce5c8a52a8b8e340ca65f13e5
+
+  - name: smbclient
+    config-opts:
+      - --without-json
+      - --without-ad-dc
+      - --without-ldap
+      - --without-ads
+      - --without-pam
+      - --disable-python
+      - --disable-cups
+      - --disable-iprint
+      - --without-systemd
+      - --bindir=.
```

Comment:
setting this and below to `.` does not seem right.

Reply:
I removed the whole block